tests/test_admin_and_files.py

```diff
@@ -27,7 +27,7 @@ def test_commentaries_registered_in_admin(self):
 
 class GitignoreTests(TestCase):
     def test_gitignore_exist(self):
-        file_exists = os.path.exists('.gitignore')
+        file_exists = os.path.exists(".gitignore")
         assert file_exists
```

Comment:
Use `self.assertTrue(file_exists)` instead of `assert file_exists` for better integration with Django's test framework.